docs/configs/bebop.py

```diff
@@ -0,0 +1,43 @@
+from parsl.addresses import address_by_hostname
+from parsl.launchers import SrunLauncher, SingleNodeLauncher
+from parsl.providers import SlurmProvider
+
+from funcx_endpoint.endpoint.utils.config import Config
+from funcx_endpoint.executors import HighThroughputExecutor
+from funcx_endpoint.strategies import SimpleStrategy
+# fmt: off
+
+# PLEASE UPDATE user_opts BEFORE USE
+user_opts = {
+    'bebop': {
+        'worker_init': '',
+        'scheduler_options': '#SBATCH --ntasks-per-node=10',
+    }
+}
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            max_workers_per_node=10,
+            strategy=SimpleStrategy(max_idletime=300),
+            address=address_by_hostname(),
+            provider=SlurmProvider(
+                'bdws',
+                launcher=SingleNodeLauncher(),
```

Comment:
This is odd, for slurm i'd expect `SrunLauncher` as the default as that would still work when `nodes_per_block>1`